webui/src/lib/components/repositoryCreateForm.jsx

```diff
@@ -110,6 +110,8 @@ export const RepositoryCreateForm = ({ config, onSubmit, onCancel, error = null,
                         e.preventDefault();
                         onCancel();
                     }}>Cancel</Button>
+                    <Button href="https://docs.lakefs.io/quickstart/repository.html#create-the-repository">More info on
+                        creating a repository in lakeFS</Button>
```

Comment:
Adding the image of this implementation so we would have a base for a discussion.

Generally the direction is really good, the wording is accurate and in the appropriate location. It is better to make this something else other than a button. I think it would be nicer to have it simply as link with text below the existing buttons in a way this is readable but also not part of the key information displayed in this form. It is less important than anything else displayed in the form and the UI should reflect that.

![image](https://user-images.githubusercontent.com/9861474/135744189-7f3bdfc6-249b-48e4-b494-5fb609eca89e.png)

Reply:
Also, clicking the link should open a different tab.